style-guide/svg/times.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="1792" height="1792" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This is the same name of the icon used on My Yoast, see `client/src/icons/times.svg`. I think we should be consistent and keep it as is or change it also on My Yoast.

Reply:
Ah, okay. Let's keep it this way 👌

Reply:
It's a fun one, I can also never find it in fontawesome because of that name 😄 it doesn't really match how the icon is being used on the web today. But while we're sourcing from fontawesome we best keep it the same yeah.